### PR TITLE
Fleshed out the README.md file.

### PR DESCRIPTION
**Pull request by kuraudoh**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,28 @@
-# BankSimulator
\ No newline at end of file
+# Overview
+Bank Simulator is a learning assignment designed to assist in teaching the developers how to code on a shared project using Git. This README.md file itself serves as a light introduction into how markdown works. For more information on markdown and its syntax, please visit [markdown.org's syntax page](https://www.markdownguide.org/basic-syntax).
```
Comment (kuraudoh) [inline README.md:2]: Bank simulator? What a shitty name. Change it.

Comment (kuraudoh) [reply README.md:2]: No. I like it. I'm the dictator here. Deal with it.

**Comment by oddsbodkin** [comment]
Man this is shit.

**Comment by NealNAL** [comment]
test

**Pull request merged**